### ISSUE TITLE
Reset back to favourites on unlock or going back home

### ISSUE THREAD

**Issue by Draic** [8bitPit/Niagara-Issues#2909]
### Is your feature request related to a problem?

when unlocking the phone or going home from an app, the launcher is in its last state which is confusing in many cases

### Describe the solution you'd like

It is honestly driving me nuts that when I go to the home screen from an app or unlock, it is stuck in the same menu it was before. Meaning the alphabet list at some location. I would like to have the reliability and confidence that my screen is in an expected state especially on unlock. That I need to be very aware of where the launcher is set to on last use is annoying to me personally and I'd like to request the option for it to reset back to favorites instead of remembering its location. 

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by TFackler** [comment]
Please take a look at this exert from our knowledge base:

### Pressing home doesn't show favorites
Suppose you launch an app list but return to Niagara Launcher immediately afterward (< 3 seconds). In that case, Niagara Launcher returns you to the last position on the app list instead of the main home screen. This is intentional and saves you time when you accidentally tap on the wrong app.

For the same reason, Niagara Launcher keeps you on the app list if you turn off your phone screen and turn it on again immediately after (e.g., by pressing the power button or using Niagara Launcher's double tap to sleep feature).

From: [Pressing home does not open app search](https://help.niagaralauncher.app/article/86-pressing-home-doesnt-show-favorites)

**tl;dr**
This is intended behavior and only happens when immediately going back home after opening and app or unlocking the device immediately after locking it.
